docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -400,6 +400,15 @@ public class MyClass {
 The URI needs to use a virtual host name (that is, a service name, not a host name).
 The Ribbon client is used to create a full physical address.
 
+IMPORTANT: If you want to use a `@LoadBalanced WebClient.Builder` it is recommended that you add the `org.springframework.cloud:spring-cloud-loadbalancer` dependency to your project.
+Then, `ReactiveLoadBalancer` will be used underneath.
+Otherwise, the request will be handled by a non-reactive `LoadBalancerClient`.
+
+TIP: The `ReactorLoadBalancer` used underneath supports caching. If `cacheManager` is detected,
+cached version of `ServiceInstanceSupplier` will be used. If not, we will retrieve instances
```

Comment:
Possibly add info about what they need to do to enable this?  Point to the docs to enable caching?